### PR TITLE
Add doc aliases script

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Linked to https://github.com/gtk-rs/gtk3-rs/pull/539 and https://github.com/gtk-rs/gtk-rs-core/pull/83.

### CODE REVIEW

File: doc_aliases.py [hunk 1]
```diff
@@ -0,0 +1,184 @@
+import os
+import sys
+
+
+IGNORE_C_FNS = ["_get_reference_count", "_status", "_free", "_destroy", "_unref", "gdk_device_free_history"]
+IGNORE_FNS = [
+    "deref", "deref_mut", "drop", "fmt", "clone", "to_value", "g_value_get_flags",
+    "g_value_set_flags", "get_type",
+]
+IGNORE_FNS_START = ["to_glib", "from_glib", "from_raw", "to_raw"]
```
Comment (bilelmoussaoui) [inline doc_aliases.py:10]:
```suggestion
IGNORE_FNS_START = ["into_glib", "from_glib", "from_raw", "to_raw"]
```
we don't have `to_glib` anymore

**Comment by bilelmoussaoui** [comment]
otherwise lgtm, let's see if it breaks any CI. Would be really nice if it can handle enum variants though. You can get them from the `IntoGlib` implementation and map them back to the enum variants I guess?

**Comment by GuillaumeGomez** [comment]
@bilelmoussaoui I guess I can, gonna add it tomorrow.

File: doc_aliases.py (hunk 1 above)
Comment (GuillaumeGomez) [reply doc_aliases.py:10]: We have it in `cairo`. ;) Gonna add `into_glib` though.

**Pull request merged**